packages/m2m-gateway/test/api/purposes/downloadPurposeVersionDocument.test.ts

```diff
@@ -0,0 +1,88 @@
+import { describe, it, expect, vi } from "vitest";
+import { generateToken } from "pagopa-interop-commons-test";
+import { AuthRole, authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { generateId } from "pagopa-interop-models";
+import { api, mockPurposeService } from "../../vitest.api.setup.js";
+import { appBasePath } from "../../../src/config/appBasePath.js";
+import {
+  purposeVersionDocumentNotFound,
+  purposeVersionNotFound,
+} from "../../../src/model/errors.js";
+import { getMockDownloadedDocument } from "../../mockUtils.js";
+import {
+  testExpectedMultipartResponse,
+  testMultipartResponseParser,
+} from "../../multipartTestUtils.js";
+
```

Comment:
I forgot to rename the files for other download endpoints in #2096 🥲
Could you rename them in this PR? Both in `integration` and in `api`
- `getAgreementConsumerDocument.test.ts` >> `downloadAgreementConsumerDocument.test.ts`
- `getEserviceDescriptorInterface.test.ts` >> `downloadEserviceDescriptorInterface.test.ts`